bundle/deploy/terraform/convert.go

```diff
@@ -275,5 +311,36 @@ func TerraformToBundle(state *tfjson.State, config *config.Root) error {
 		}
 	}
 
+	for _, src := range config.Resources.Jobs {
+		if src.ModifiedStatus == "" && src.ID == "" {
+			src.ModifiedStatus = resources.ModifiedStatusCreated
```

Comment:
Can we set it as a result of convRemoteToLocal call? If the resource is in remote but not local state it means it's created, correct?